packages/media-fields/src/media_thumbnail/view.tsx

```diff
@@ -17,10 +18,44 @@ import type { DataViewRenderFieldProps } from '@wordpress/dataviews';
 import { getMediaTypeFromMimeType } from '../utils/get-media-type-from-mime-type';
 import type { MediaItem } from '../types';
 
+function FallbackView( {
```

Comment:
So this doesn't really add much to what was already there, it just forks when there's an image load error.

Reply:
I like it as a separate component, though, it improves readability 👍